src/atoms/loader/Loader.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+ 
+const Loader = ({loadingText}) => {
+   return (
+       <h1>{loadingText}</h1>
+   );
+};
+
+Loader.propTypes = {
+    loadingText: PropTypes.string
+  };
+  
+Loader.defaultProps = {
+    loadingText:''
```

Comment:
So now if we have used default props, then we can make use of it in appropriate way like : Loading Data ... otherwise if it still comes as undefined we do not show anything which is similar to showing empty string.